siliconcompiler/report/streamlit_viewer.py

```diff
@@ -19,35 +19,31 @@
 SUCCESS_COLOR = '#8EA604'  # green
 PENDING_COLOR = '#F5BB00'  # yellow, could use: #EC9F05
 FAILURE_COLOR = '#FF4E00'  # red
-
-sc_logo_path = os.path.join(os.path.dirname(__file__), '..', 'data', 'logo.png')
-
-sc_font_path = \
+SC_LOGO_PATH = os.path.join(os.path.dirname(__file__), '..', 'data', 'logo.png')
+SC_FONT_PATH = \
     os.path.join(os.path.dirname(__file__), '..', 'data', 'RobotoMono', 'RobotoMono-Regular.ttf')
-
-sc_about = [
+SC_ABOUT = [
     f"SiliconCompiler {sc_version}",
     '''A compiler framework that automates translation from source code to
      silicon.''',
     "https://www.siliconcompiler.com/",
     "https://github.com/siliconcompiler/siliconcompiler/"
 ]
-
-sc_menu = {"Get help": "https://docs.siliconcompiler.com/",
+SC_MENU = {"Get help": "https://docs.siliconcompiler.com/",
            "Report a Bug":
            '''https://github.com/siliconcompiler/siliconcompiler/issues''',
-           "About": "\n\n".join(sc_about)}
+           "About": "\n\n".join(SC_ABOUT)}
 
 # opened by running command in siliconcompiler/apps/sc_dashboard.py
-parser = argparse.ArgumentParser('dashboard')
-parser.add_argument('cfg', nargs='?')
-args = parser.parse_args()
+PARSER = argparse.ArgumentParser('dashboard')
+PARSER.add_argument('cfg', nargs='?')
+ARGS = PARSER.parse_args()
```

Comment:
these should not be all caps
The all caps is meant for static information

Reply:
Yeah, I was going back and forth on this. I realize the argument that the variable is not static from run to run, but it is static for the duration of any given run. I'm seeing that `PARSER` is updated in line 39, so that would not be static given either definition, but `ARGS` and some of the other variables I capitalized are. To me, the second argument makes more sense, but I can change it back if that's not standard practice.

Reply:
To me all caps implies it's a constant for everything (see the strings and color defs you have), these are not constant and should not be caps